addon/components/pie-chart.js

```diff
@@ -162,11 +162,17 @@ export default ChartComponent.extend(FloatingTooltipMixin,
   }),
 
   otherData: Ember.computed('sortedDataWithOther.[]', 'sortFunction', function() {
-    var _ref;
     var otherSlice = _.find(this.get('sortedDataWithOther'), function(d) {
       return d._otherItems;
     });
-    var otherItems = (_ref = otherSlice != null ? otherSlice._otherItems : void 0) != null ? _ref : [];
+
+    var otherItems;
+    if (otherSlice != null && otherSlice._otherItems != null) {
```

Comment:
That goes for pretty much everywhere in this codebase. I'd like to update that at some point as well, but for now I'd rather err on the side of caution.